winterfell/middlewares/passport.js

```diff
@@ -48,4 +55,25 @@ module.exports = function (app) {
             }
         });
     }
+
+    passport.use('idme', new OAuth2Strategy({
+        authorizationURL: 'https://api.id.me/oauth/authorize',
+        tokenURL: 'https://api.id.me/oauth/token',
+        clientID: idmeClientId,
+        clientSecret: idmeSecretId,
+        callbackURL: '/auth/idme/callback'
```

Comment:
hmm it worked when I tried it locally. But I think you're right, it should be a full url.